src/app/page.tsx

```diff
@@ -1,7 +1,19 @@
 export default function Home() {
   return (
-    <main className="flex min-h-screen flex-col items-center justify-between p-24">
+    <main className="centerText page-landing-background">
```

Comment:
[Comment]: One thing I noticed looking at the page is that the text color needs to white to work with the dark background. This is part of #10 so will be taken care of, but noting it here.

Reply:
Placed temporary color scheme to text so you can see the text, Is to be replaced once global theme is done. @Nqnt41 @